scss/detail.scss

```diff
@@ -1,12 +1,28 @@
-.master-container{
+.master-container {
   display: flex;
+  flex-direction: column;
   height: 768px;
   width: 1366px;
   background-color: $wild-sand;
   justify-content: center;
   align-items: center;
+
+  .return-button {
+    align-self: flex-start;
+    color: $black;
+    font-size: 20px;
+    line-height: 24px;
+    margin-bottom: 108px;
+    margin-left: 233px;
+  }
+
+  .return-button::before {
```

Comment:
For pseudo-elements, use nesting and `&` operator:

```scss
.return-button {
  ...
  &::before {
    ...
  }
}
```

Reply:
All changes made as requested!!